pyvista/core/datasetattributes.py

```diff
@@ -246,7 +249,7 @@ def __delitem__(self, key: str):
 
     def __contains__(self, name: str) -> bool:
         """Implement the ``in`` operator."""
-        return name in self.keys()
+        return name in self
```

Comment:
This is from `SIM118` in [`flake8-simplify-sim`](https://beta.ruff.rs/docs/rules/#flake8-simplify-sim).
This class is not a dictionary and does not work well. I would cheat use the following expression.
```suggestion
        keys = self.keys()
        return name in keys
```